src/compose.js

```diff
@@ -6,6 +6,9 @@ var reverse = require('./reverse');
  * Performs right-to-left function composition. The rightmost function may have
  * any arity; the remaining functions must be unary.
  *
+ * The result of compose is not automatically curried so functions with optional
+ * arguments can be used as the rightmost function and are called immediately.
+ *
```

Comment:
In fitting with the bare-bones documentation approach I've used your example @CrossEye